pkg/local_object_storage/blobovnicza/get_test.go

```diff
@@ -0,0 +1,75 @@
+package blobovnicza
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	oidtest "github.com/nspcc-dev/neofs-sdk-go/object/id/test"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBlobovnicza_Get(t *testing.T) {
+	t.Run("re-configure object size limit", func(t *testing.T) {
```

Comment:
You have a single `t.Run` in this test, why not just change the name of the test?

Reply:
I stick to the approach of creating a test function for an operation, not for a scenario. Do u think it's worth to have a separate function?

Reply:
I mean we don't yet have other tests, so this looks a bit strange.